README.md

```diff
@@ -7,3 +7,103 @@ This is the API of [metalstack.cloud](https://metalstack.cloud).
+    max_len: 128
+  }];
+  string description = 4 [(buf.validate.field).string = {max_len: 128}];
+  repeated string tags = 8 [(buf.validate.field).repeated.max_items = 100];
+  bool static = 9;
+}
+```
+
+In this example we can see the motivation behind the method options.
+
+1. Get: can be issued by project owner, editor, viewer and is excluded from auditing
+2. Allocate: can be used by project owner, editor and is an service-method which requires deposited payment information
+3. Both methods are project-scoped, since they are annotated by a project role -> Request object needs to have the **project** field in order to specify the target project of the service method
+
+Further explanations are explained in the following:
+
+### Auth:
+
+These options specify the RBAC of the api-endpoint.
+
+| Option         | Description                                  | Values      | Explanation                                           |
+| -------------- | -------------------------------------------- | ----------- | ----------------------------------------------------- |
+| TENANT_ROLE\_  | Specifies the required tenant role           | UNSPECIFIED |                                                       |
+|                |                                              | OWNER       | tenant owner                                          |
+|                |                                              | EDITOR      | tenant editor                                         |
+|                |                                              | VIEWER      | tenant viewer                                         |
+|                |                                              | GUEST       | tenant guest                                          |
+| PROJECT_ROLE\_ | Specifies the required project role          | UNSPECIFIED |                                                       |
+|                |                                              | OWNER       | project owner                                         |
+|                |                                              | EDITOR      | project editor                                        |
+|                |                                              | VIEWER      | project viewer                                        |
+| ADMIN_ROLE\_   | Specifies the required admin role            | UNSPECIFIED |                                                       |
+|                |                                              | EDITOR      | admin editor                                          |
+|                |                                              | VIEWER      | admin viewer                                          |
+| VISIBILITY\_   | Specifies the visibility of the api-endpoint | UNSPECIFIED |                                                       |
+|                |                                              | PUBLIC      | api-method is visbile to public and requires no token |
```

Comment:
I think `requires no token` is a bit misleading towards the direction that it would be forbidden. What about `no token is required` or something similar?